tests/migrations/test_optimizer.py

```diff
@@ -212,16 +212,7 @@ def test_optimize_through_create(self):
             ],
             [],
         )
-        # This should not work - FK should block it
-        self.assertDoesNotOptimize(
-            [
-                migrations.CreateModel("Foo", [("name", models.CharField(max_length=255))]),
-                migrations.CreateModel("Bar", [("other", models.ForeignKey("testapp.Foo", models.CASCADE))]),
-                migrations.DeleteModel("Foo"),
-            ],
-        )
-        # The same operations should be optimized if app_label is specified and
-        # a FK references a model from the other app.
```

Comment:
Moved just below without `testapp` specified which doesn't make sense when `app_label` is not specified.